storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc

```diff
@@ -0,0 +1,39 @@
+[id="persistent-storage-csi-google-cloud-file"]
+= Google Filestore CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-google-cloud-file
+
+toc::[]
+
+[id="persistent-storage-csi-google-cloud-file-overview"]
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Filestore Storage.
+
+:FeatureName: Google Filestore CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Google Filestore Storage assets, {product-title} installs the Google Filestore CSI Driver Operator and the Google Filestore CSI driver in the `openshift-cluster-csi-drivers` namespace.
+
+* The _Google Filestore CSI Driver Operator_ does not provide a storage class by default, but xref:../../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file-create-sc_persistent-storage-csi-google-cloud-file[you can create one if needed]. The Google Filestore CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on demand, eliminating the need for cluster administrators to pre-provision storage.
+
+* The _Google Filestore CSI driver_ enables you to create and mount Google Filestore PVs.
+
+[NOTE]
+====
+Provisioning a cluster may take up to five minutes.
+====
+
+You need to manually delete Google Filestore instances created by the operator, since instances are not deleted when the cluster is destroyed.
```

Comment:
`created by the CSI driver` (the operator doesn't create volumes, it just installs the CSI driver.